manifests/init.pp

```diff
@@ -78,4 +78,18 @@
     group   => '0',
   }
 
+  if ($facts['os']['release']['major'] < '8' and $facts['os']['family'] == 'RedHat') {
+    ## ADD /etc/motd.d/* SUPPORT TO RHEL7, ETC
+    File {
+      mode   => '0644',
+    }
+    file { '/etc/profile.d/motd.csh':
+      source => "puppet:///modules/${module_name}/etc/profile.d/motd.csh",
+    }
+    file { '/etc/profile.d/motd.sh':
+      source => "puppet:///modules/${module_name}/etc/profile.d/motd.sh",
```

Comment:
The comment suggests files go in `/etc/motd.d/` but the code puts them in `/etc/profile.d/`.

Which one is correct?

Reply:
Understood, I see it now.